### PR TITLE
boards: arm: stm32_min_dev: Minimal enhancements

### PR DESCRIPTION
**Pull request by gon1332**
Some minor enhancements will follow.

Signed-off-by: Ioannis Konstantelias <ikonstadel@gmail.com>

### CODE REVIEW

File: boards/arm/stm32_min_dev/doc/index.rst [hunk 1]
```diff
@@ -43,6 +43,18 @@ Additional device support is left for the user to implement.
 More information on hooking up peripherals and lengthy how to articles can be
 found at `EmbedJournal`_.
 
+The pinout diagram of STM32 Minimum Deveopment Blue Pill board can be seen
```
Comment (dbkinder) [inline boards/arm/stm32_min_dev/doc/index.rst]: s/Deveopment/Development/
s/bellow/below/

**Review by dbkinder — CHANGES_REQUESTED** [review]
a couple of misspellings...

**Comment by gon1332** [comment]
@dbkinder Do you want me to squash the commits?

**Review by dbkinder — APPROVED** [review]
OK for the docs, but curious why you removed the links to the blue/black pill boards? Oh, and yes, project convention would be to squash commits like these.

**Comment by gon1332** [comment]
> OK for the docs, but curious why you removed the links to the blue/black pill boards?

They are not working anymore. I tried to find something similar but with no success.

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**